builder/builder-next/worker/containerdworker.go

```diff
@@ -0,0 +1,114 @@
+package worker
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/content"
+	"github.com/moby/buildkit/cache"
+	"github.com/moby/buildkit/client"
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/executor"
+	"github.com/moby/buildkit/exporter"
+	"github.com/moby/buildkit/frontend"
+	"github.com/moby/buildkit/session"
+	"github.com/moby/buildkit/solver"
+	"github.com/moby/buildkit/worker/base"
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// ContainerdWorker is a local worker instance with dedicated snapshotter, cache, and so on.
+type ContainerdWorker struct {
+	baseWorker *base.Worker
```

Comment:
No biggie, just wondering: shouldn't it suffice to embed `*base.Worker` and "inherit" its methods, so you only override the `Exporter` method, no need for the rest of the one-liners.